patches/react-native-vision-camera/details.md

```diff
@@ -0,0 +1,9 @@
+# `react-native-vision-camera` patches
+
+### [react-native-vision-camera+4.6.1.patch](react-native-vision-camera+4.6.1.patch)
+
+- Reason: Fixes an issue in VisionCamera where the `CameraSession` doesn't get de-initialized and recycled when the screen gets popped/unmounted from the Navigation stack in `@react-navigation/native-stack`.
+- Upstream PR/issue: N/A (This will be fixed once VisionCamera is migrated to Nitro Views)
```

Comment:
Is there a planning issue for this migration? could we link it here and explain?